tasks/restore.yml

```diff
@@ -1,23 +1,47 @@
 ---
+- block:
+    - name: Sync etcd backup file {{ restore_file_name }} from localhost to all etcd nodes
```

Comment:
If "restore_file_name" is not defined, the name displayed during the ansible run will have "{{ restore_file_name }}" in it instead of the value.
To avoid that I would make the name something like this:
"Sync the user-defined etcd backup file from localhost to all etcd nodes"